frontend/src/features/projects/dispose/hooks/useStepForm.tsx

```diff
@@ -71,6 +71,7 @@ const useStepForm = () => {
       .catch((error: AxiosError) => {
         const msg: string = error?.response?.data?.error ?? error.toString();
         formikRef?.current?.setStatus({ msg });
+        throw Error('axios request failed');
```

Comment:
Ensures that add/update failures prevent navigation to the next step (hiding the API error).